apis/config/v1alpha1/projectconfig_types.go

```diff
@@ -7,6 +7,59 @@ import (
 	apiv1alpha1 "github.com/os-observability/tempo-operator/apis/tempo/v1alpha1"
 )
 
+// BuiltInCertManagement is the configuration for the built-in facility to generate and rotate
+// TLS client and serving certificates for all Tempo services and internal clients except
+// for the tempo-gateway.
+type BuiltInCertManagement struct {
+	// CACertValidity defines the total duration of the CA certificate validity.
+	CACertValidity string `json:"caValidity,omitempty"`
+	// CACertRefresh defines the duration of the CA certificate validity until a rotation
+	// should happen. It can be set up to 80% of CA certificate validity or equal to the
+	// CA certificate validity. Latter should be used only for rotating only when expired.
+	CACertRefresh string `json:"caRefresh,omitempty"`
```

Comment:
The same comment as previous one:
* could a duration be used as a type? 
* what is the default? 

also the same comment applies for other fields in this file.